src/services/telemetry/TelemetryService.ts

```diff
@@ -396,6 +507,7 @@ export class TelemetryService {
 			...this.telemetryMetadata,
 		}
 
+		this.userId = userInfo.id
```

Comment:
Since we only want to identify user once afaik, maybe it'd be helpful to check if this.userId = user info.id to identify the same user multiple time?